tests/vps.nix

```diff
@@ -122,13 +126,17 @@
       backup.succeed("cp -R /etc/ahayzen.com/restore/fixtures/* /tmp/restore-root/")
       backup.succeed("chown -R 2000:2000 /tmp/restore-root/")
 
-      # Check files exist
+      # Check files exist and permissions are correct
       backup.succeed("test -d /tmp/restore-root/test-page/docker-compose-runner/wagtail-ahayzen/db")
       backup.succeed("test -e /tmp/restore-root/test-page/docker-compose-runner/wagtail-ahayzen/db/db.sqlite3")
+      backup.succeed("ls -nd /tmp/restore-root/test-page/docker-compose-runner/wagtail-ahayzen/db/db.sqlite3 | awk 'NR==1 {if ($3 == 2000) {exit 0} else {exit 1}}'")
 
       # Run the restore
       backup.succeed("/etc/ahayzen.com/restore.sh vps headless@vps /tmp/restore-root/test-page")
```

Comment:
even with `sudo`, `--super`, and `--numeric-ids` etc the permissions end up incorrect after a restore, the final files are `root` rather than `2000` or `unpriv` :shrug:

Reply:
Instead creating a tar archive and then rsync'd this may be a better route